auxprogs/filterBam/src/Makefile

```
@@ -28,10 +28,10 @@ CHECKBAM:
 	fi
 
 $(PROGRAM) : $(OBJECTS)
-	$(LINK.cc) $(CFLAGS) $(LDFLAGS) $^ -o $@ $(LIBS) 
+	$(LINK.cc) $(CXXFLAGS) $(LDFLAGS) $^ -o $@ $(LIBS)
```

```suggestion
	$(LINK.cc) $(CXXFLAGS) $(CPPFLAGS) $(LDFLAGS) $^ -o $@ $(LIBS)
```
- add $(CPPFLAGS)  ?

---

not required